00_Base/src/mapper/CitrineOcpiLocationMapper.ts

```diff
@@ -105,9 +112,10 @@ export class CitrineOcpiLocationMapper implements IOcpiLocationMapper {
           ];
 
         if (!ocpiEvseInfo) {
-          throw new NotFoundException(
-            `OCPI EVSE ${UID_FORMAT(evseAttributes.station_id, evseAttributes.id)} does not exist.`,
+          this.logger.warn(
+            `OCPI EVSE ${UID_FORMAT(evseAttributes.station_id, evseAttributes.id)} does not exist - will skip.`,
```

Comment:
Are all caps method names like `UID_FORMAT` usually a thing? 🤔 

I feel like that's typically for final vars only.

Also, I am questioning if this method should really live instead EvseDTO or if it is better suited for this mapper class.

Reason being that then we avoid having to put OCPI 2.0.1 references inside the EvseDTO.

Reply:
i was using this to indicate it's a static method but can change the naming convention based on what we decide 👍 the general placement of this method is indeed a bit strange... i think i'll move this into its own dedicated util or const file 🤔